pkg/cluster/controllers/storage/tenant.go

```diff
@@ -215,7 +215,7 @@ func (m *MinIOController) generateMinIOCR(ctx context.Context, harborcluster *go
 					Servers:             harborcluster.Spec.InClusterStorage.MinIOSpec.Replicas,
 					VolumesPerServer:    harborcluster.Spec.InClusterStorage.MinIOSpec.VolumesPerServer,
 					VolumeClaimTemplate: m.getVolumeClaimTemplate(harborcluster),
-					Resources:           *m.getResourceRequirements(harborcluster),
+					Resources:           harborcluster.Spec.InClusterStorage.MinIOSpec.Resources,
```

Comment:
we will not define default resource?

Reply:
Why do we need to define the default resource for the minio pod when the customers don't want the limited resources used by it?